bots/tests-scan

```diff
@@ -116,7 +122,7 @@ def main():
                         help='Display human readable output rather than tasks')
     parser.add_argument('-d', '--dry', action="store_true", default=False,
                         help='Don''t actually change anything on GitHub')
-    parser.add_argument('--repo', default=None,
+    parser.add_argument('--repo', default="cockpit-project/cockpit",
```

Comment:
Conceptually tests-scan and tests-trigger should not have a default repo, but use `GitHub.__init__()`'s logic of figuring it out from your current `origin`. Then tests-trigger Just Works™ with the bots/ checkout in e. g. welder-web or podman. Also, we try to remove cockpit special cases, let's not introduce more of them. Use `api.repo` instead of `opts.repo` to refer to the actual repo then.

However, I realize that this mostly applies to tests-trigger, less for tests-scan. Right now tests-scan is pretty much exclusively used from cockpit, and it's actively broken when you call it from e. g. a welder-web checkout. But let's fix this. :-)

Reply:
> Also, we try to remove cockpit special cases, let's not introduce more of them.

Yes, I am fully aware I introduced one such case. But now it works pretty much like that - that if you don't provide repo, c-p/cockpit is assumed. And since I din't want to break how it works now with this PR I went with this approach.

> Use api.repo instead of opts.repo to refer to the actual repo then.

Ha, that's just what I needed. Thanks!

Reply:
> But now it works pretty much like that - that if you don't provide repo, c-p/cockpit is assumed. 

So "now" means "with this PR", as with now==master this isn't true; but as I said, running tests-scan from e. g. welder-web doesn't work very well, so on second thought I agree -- let's keep the explicit cockpit default here, as scanning tests  from that should work and cover all external projects.